shared/macros-oval.jinja

```diff
@@ -250,6 +251,69 @@
   </unix:file_object>
 {{%- endmacro %}}
 
+{{#
+  Macro to define the OVAL test to check if there is a line in file with a pair of argument=value (Criterion definition).
+  Parameters:
+    - filepath (String): Path to the file to be checked.
+    - name (String): Argument name
+    - value (String): Argument value
+    - application (String): The application which the configuration file is being checked. Can be any value and does not affect the actual OVAL check.
+#}}
+{{%- macro oval_argument_in_file_criterion(filepath, name, value, application='') -%}}
+{{%- set name_value = name+"="+value -%}}
+  <criterion comment="Check if argument {{{ name_value }}}{{% if application %}} for {{{ application }}}{{% endif %}} is present in {{{ filepath }}}" 
+  test_ref="test_{{{ rule_id }}}_{{{ name_value|escape_id }}}_argument_in_{{{ filepath|escape_id }}}"/>
+{{%- endmacro -%}}
+
+{{#
+  Macro to define the OVAL test to check if there is a line in file with a pair of argument=value (Test definition).
+  Parameters:
+    - filepath (String): Path to the configuration file to be checked.
```

Comment:
I would add here that the operation in `filepath` is always a `pattern match`

Reply:
Done